guide/contribute/review.md

```diff
@@ -0,0 +1,102 @@
+
+Pull requests marked as **WIP (Work in progress)** usually require structural feedback and do not need a thorough evaluation. If the WIP pull request description doesn't specify the required feedback type, feel free to ask the author by leaving a comment.
+
+### Reviewing pull requests
+
+From the list of pull requests select the one you want to review. Read the description of a pull request, and if it references an issue, check the issue to get the context. To begin the review process click on the `Files Changed` tab.
+
+The tab is split into two sections. The section on the left represents the current code/content. The section on the right represents the proposed changes.
+
+To begin reviewing, browse through the changed files from top to bottom. When you identify something that requires a comment or suggestion, click on the `+` icon in front of the line you want to comment on, and toggle the comment window and leave your comment.
+
+{% include picture.html
+   image = "/assets/images/guide/contribute/review/review-pull-request-comment.gif"
+   retina = "/assets/images/guide/contribute/review/review-pull-request-comment.gif"
+   mobile = "/assets/images/guide/contribute/review/review-pull-request-comment.gif"
+   mobileRetina = "/assets/images/guide/contribute/review/review-pull-request-comment.gif"
+   alt-text = "pull-request-comment"
+   width = 1438
+   height = 709
+%}
+
+If you have a concrete `suggestion` for a change, you can make a suggestion. To make a suggestion click on the suggestion icon, or use `Ctrl+G` or `cmd+G`. If your direct suggestion is accepted, it will count as a commit and you will be credited as a co-author of the commit.
+
+{% include picture.html
+   image = "/assets/images/guide/contribute/review/review-pull-request-suggest.gif"
+   retina ="/assets/images/guide/contribute/review/review-pull-request-suggest.gif"
+   mobile ="/assets/images/guide/contribute/review/review-pull-request-suggest.gif"
+   mobileRetina ="/assets/images/guide/contribute/review/review-pull-request-suggest.gif"
+   alt-text = "pull-request-comment"
+   width = 1438
+   height = 709
+%}
+
+When you're done with leaving individual feedback, you can submit it. Click on the `Leave a review` button in the upper right corner. There are three types of feedback you can leave:
+
+- **Comment** - general feedback without explicit approval.
+- **Approve changes** - feedback approving the proposed changes.
+- **Request changes** - feedback explicitly suggesting changes.
+
+#### Best practices when reviewing pull requests
+
+**Do**
+
+- Leave a suggestion instead of general feedback whenever possible, especially for typo and grammar-related feedback
+- Be humble and humble when reviewing
```

Comment:
`humble and humble`?

Reply:
Double humbleness! Any other more appropriate word, or should I just say be humble when reviewing?

Reply:
How about `humble and constructive`?

Reply:
```suggestion
- Be humble and constructive when reviewing
```